world-server/src/world/player/PlayerHandlers.ts

```diff
@@ -26,6 +26,9 @@ export class PlayerHandler implements Messages.MsgHandler {
             Logger.warn(`Received player message does not extend PlayerMessage`);
             return;
         }
+        console.log("some message" + message.type);
```

Comment:
Remove calls to `console.log` used for debugging

Reply:
Fixed it for them :)